package.json

```diff
@@ -165,7 +165,7 @@
     "build:cjs": "tsc --project tsconfig-cjs.json && echo '{ \"type\": \"commonjs\" }' > dist/cjs/package.json",
     "build:esm": "tsc --project tsconfig-esm.json",
     "build:css": "./scripts/build-css.sh ./src/style.css ./src/style.module.css",
-    "lint": "eslint .",
+    "lint": "biome check",
```

Comment:
Would it help to add a separate `biome check --write` script?

The same can be achieved with `pnpm run lint --write`

Reply:
Yeah I prefer adding the `--write` when necessary...